Update chemistry conceptual docs for v0.7 (#427)
MicrosoftDocs/quantum-docs-pr: articles/libraries/chemistry/concepts/symmetries-molecular.md

Molecular orbital integrals (i.e. the $h\_{pq}$ and $h\_{pqrs}$ terms) such as these are represented using the `OrbitalIntegral` type, which provides a number of helpful functions to express this symmetry.
```csharp

>     // We load the namespace containing orbital integral objects.

Reviewer: ```suggestion
    // Load the namespace containing orbital integral objects.
```

    using Microsoft.Quantum.Chemistry.OrbitalIntegrals;

>     // We create a `OrbitalIntegral` object to store a one-electron molecular 

Reviewer: ```suggestion
    // Create a `OrbitalIntegral` object to store a one-electron molecular 
```

Rather than constructing a Fermionic Hamiltonian by adding `FermionTerm`s, we may directly add all terms corresponding to each orbital integral automatically. For example, the following code automatically enumerates over all permutational symmetries and orders the terms in canonical order: 
```csharp

>     // We load the namespace containing fermion objects. This

Reviewer: ```suggestion
    // Load the namespace containing fermion objects. This
```

    // example also uses LINQ queries.
    using Microsoft.Quantum.Chemistry.Fermion;
    using System.Linq;

    // We create a `OrbitalIntegral` object to store a two-electron molecular 
    // orbital integral data.
    var orbitalIntegral = new OrbitalIntegral(new[] { 0, 1, 2, 3 }, 0.123);

>     // We create an `OrbitalIntegralHamiltonian` object to store the orbital integral

Reviewer: ```suggestion
    // Create an `OrbitalIntegralHamiltonian` object to store the orbital integral
```

    // terms
    var orbitalIntegralHamiltonian = new OrbitalIntegralHamiltonian();
    orbitalIntegralHamiltonian.Add(orbitalIntegral);

>     // We now convert the orbital integral representation to a fermion

Reviewer: ```suggestion
    // Convert the orbital integral representation to a fermion
```

    // representation. This also requires choosing a convention for 
    // mapping spin orbital indices to integer indices.
    var fermionHamiltonian = orbitalIntegralHamiltonian.ToFermionHamiltonian(IndexConvention.UpDown);

>     // Alternatively, we we add orbital integrals directly to a fermion Hamiltonian

Reviewer: ```suggestion
    // Alternatively, one can add orbital integrals directly to a fermion Hamiltonian
```

    // as follows. This automatically enumerates over all symmetries, and then

>     // orders the `HermitianFermionTerm` instances in canonical order. We will need to

Reviewer: Here as in the above, it would be better to use passive voice and avoid "We". Such as "An indexing convention etc."
